src/gocept/selenium/scripts/tests/test_converthtmltests.py

```diff
@@ -1,6 +1,6 @@
 from io import StringIO
 import unittest
-import mock
+from unittest import mock
```

Comment:
Hu, we never declared mock as dependency before...

Reply:
I see.